train.py

```diff
@@ -0,0 +1,231 @@
+"""
```

Comment:
train.py is depreciated, we use only train_runnable now. Nice if you delete this